Scripts/build.swift

```diff
@@ -23,49 +23,61 @@ enum TaskError: Error {
 enum Platform: String, CaseIterable, CustomStringConvertible {
     case iOS_12
     case iOS_13
+    case iOS_14
```

Comment:
while I was here I added support for testing on iOS 14. I should add iOS 15 too but... for another day